plugins/calibre.koplugin/search.lua

```diff
@@ -7,6 +7,7 @@ local ConfirmBox = require("ui/widget/confirmbox")
 local DataStorage = require("datastorage")
 local Device = require("device")
 local FileManagerBookInfo = require("apps/filemanager/filemanagerbookinfo")
+local FileManagerUtil = require("apps/filemanager/filemanagerutil")
```

Comment:
Somehow we used lowercase everywhere
https://github.com/koreader/koreader/blob/e7e0d2edb6af95bced95ce88d90a9e08f49fcf7a/frontend/apps/filemanager/filemanager.lua#L35

Reply:
I'm okay with being the odd one out ;o).

Reply:
There's some method to the naming style. FileManagerUtil implies something kinda OOP-y, filemanagerutil implies just a bunch of functions. This one perhaps lies somewhere in the middle, but note that internally it also uses filemanagerutil so I'd rather preserve easy greppability than set trends if it's all the same. ;-)

Reply:
Fair enough ;).